File: forge/src/main/java/com/selfdot/cobblemontrainers/forge/SetPermissionValidatorRunnable.java
package com.selfdot.cobblemontrainers.forge;

import com.selfdot.cobblemontrainers.CobblemonTrainers;
import net.minecraftforge.fml.DistExecutor;

public class SetPermissionValidatorRunnable implements DistExecutor.SafeRunnable {

    @Override
    public void run() {
        CobblemonTrainers.INSTANCE.setPermissionValidator_(new ForgePermissionValidator());
    }

}

File: forge/src/main/kotlin/com/selfdot/cobblemontrainers/forge/SetPermissionValidatorRunnable.kt
package com.selfdot.cobblemontrainers.forge

import com.selfdot.cobblemontrainers.CobblemonTrainers
import net.minecraftforge.fml.DistExecutor

class SetPermissionValidatorRunnable : DistExecutor.SafeRunnable {
    override fun run() {
        CobblemonTrainers.permissionValidator = ForgePermissionValidator()
    }

}
